version/version.go

```diff
@@ -0,0 +1,5 @@
+package version
+
+var (
+	Version = "0.0.1"
```

Comment:
Do we really need this extra `version` package? Isn't the one located at `pkg/version` enough?

Reply:
Removed